### ISSUE TITLE
Remove user-visible moderation log

### ISSUE THREAD

**Issue by zorbathut** [themotte/tildes#6]
We'll probably put some of it back at some point, but right now it's just "get rid of it".

**Comment by zorbathut** [comment]
Some extra notes:

I vaguely remember that Tildes has a central place where moderation actions are displayed, so that users can audit mod actions. For a variety of reasons we don't want this unless there's some way to filter them.

I also vaguely remember that I might be wrong about this, and it might be some other site that has this issue.

So this bug is "figure out if Tildes has a public moderation log, then if so, disable it". I'd recommend starting by looking for a place in the database where moderation actions are logged, then it should be easy enough to see if they're displayed anywhere.

**Comment by wbruntra** [comment]
I haven't been able to find a centralized place where users can view a log of moderation actions. Since that's not very helpful, here are notes on a few things I *have* found.

- `Log`, `LogTopic`, and `LogComment` (in `models/log/log.py`) are models used for logging actions. 
- `LogEventType` (in `enums.py`) is an enum describing the types of loggable events. Nothing jumps out as obviously related to moderator-specific actions, e.g. banning a user.
- When comments are removed, in `views/api/web/comment.py`, a `LogComment` entry is created while executing the route (the function is `put_comment_remove`).
- In contrast, on the route where a user in banned, in `views/api/web/user.py`, no log action is created, so I wonder: are these actions logged?
- The only instance I am able to find in which logs are queried *at all* is in `views/topic.py`, line 458: ```request.query(LogTopic)```. This is called when rendering the main page for a topic, and shows events such as `TOPIC_LOCK`. Removing this query seems straightforward. 
- I don't see any queries for other types of logs, i.e. `Log` or `LogComment`, which queries I believe would be necessary for logs to be user-viewable.

Not being completely familiar with this codebase, I can't totally rule out that moderator actions are logged/viewable in some other way I'm not seeing, but, well... I'm not seeing it.

Is an example of moderation logs available on https://tildes.net/ ?

**Comment by zorbathut** [comment]
Well, that looks pretty legit to me.

> Is an example of moderation logs available on https://tildes.net/ ?

I went to look and couldn't find one, but I also don't have a working login and I was thinking it was hidden behind a login gate. In retrospect, I'm pretty sure this was just some other site that I was thinking of, and I got them confused, so this was entirely my fault.

The irony is that now I think we actually *need* some kind of moderator log, just because it's useful for figuring things out internally, but I've made a new task for that: https://github.com/themotte/tildes/issues/45
